File: scripts/fix_duplicate.js
const fs = require('fs');

// Read the server.js file
const content = fs.readFileSync('server.js', 'utf-8');
const lines = content.split('\r\n');

// Remove lines 656-676 (0-indexed: 655-675) - the duplicate endpoint
const beforeDuplicate = lines.slice(0, 656);
const afterDuplicate = lines.slice(677);
const fixedLines = beforeDuplicate.concat(afterDuplicate);

// Write back
fs.writeFileSync('server.js', fixedLines.join('\r\n'), 'utf-8');

console.log('✅ Duplicate endpoint removed successfully!');
console.log('Removed lines 657-677 (duplicate /api/dashboard/running-meetings)');
console.log('Total lines before:', lines.length);
console.log('Total lines after:', fixedLines.length);

File: scripts/fix_duplicate.py
#!/usr/bin/env python3
"""
Script to remove duplicate /api/dashboard/running-meetings endpoint
"""

with open('server.js', 'r', encoding='utf-8') as f:
    lines = f.readlines()

# Find and remove lines 657-677 (duplicate endpoint)
# Keep lines before 657 and after 677
output_lines = lines[:656] + lines[677:]

with open('server.js', 'w', encoding='utf-8') as f:
    f.writelines(output_lines)

print("Duplicate endpoint removed successfully!")
print("Lines 657-677 were deleted (duplicate /api/dashboard/running-meetings)")
